packages/sdks/mitosis.config.cjs

```diff
@@ -540,6 +540,44 @@ const ANGULAR_INITIALIZE_PROP_ON_NG_ONINIT = () => ({
   },
 });
 
+const ANGULAR_FIX_AB_TESTS_INITIALIZATION = () => ({
+  json: {
+    pre: (json) => {
+      if (json.name === 'ContentVariants') {
+        if (json.state['defaultContent']) {
+          json.state['getdefaultContent'] = {
+            code: json.state['defaultContent'].code.replace('get ', 'get'),
+            type: 'method',
+          };
+          json.state['defaultContent'] = {
+            code: 'null',
+            type: 'property',
+          };
+
+          json.hooks.onUpdate.push({
+            code: `if (props.content) { state.defaultContent = state.getdefaultContent(); }`,
+          });
+        }
+      }
+    },
+  },
+});
+
+const ANGULAR_WRAP_SYMBOLS_FETCH_AROUND_CHANGES_DEPS = () => ({
```

Comment:
its for fixing the infinite loop that we go into when `contentToUse` is undefined and we still want to fetch

Reply:
why is this in a plugin? can't it be in an `onUpdate` + `useTarget()` that only has code for Angular?

Reply:
we need access to `changes` object from `ngOnChanges`